lich.rbw

```diff
@@ -2135,6 +1996,7 @@ module Games
     require_relative("./lib/weapon.rb")
     require_relative("./lib/infomon/infomon.rb")
     require_relative("./lib/stats/stats.rb")
+    require_relative("./lib/stats/spells.rb")
```

Comment:
this should probably exist at `lib/spells.rb` since they aren't exactly a character stat, but I don't know, could go either way.

Reply:
I can go either way as well - you probably saw the discussion with Tysong on it.  After we're done with the code, let me know and I'll put it in final place.  I have no real concern either way.